pkgs/applications/version-management/forgejo/default.nix

```
@@ -1,25 +1,72 @@
-{ fetchurl, gitea, lib }:
+{ lib
+, stdenv
+, buildGoPackage
+, fetchurl
+, makeWrapper
+, git
+, bash
+, gzip
+, openssh
+, pam
+, pamSupport ? true
+, sqliteSupport ? true
+}:
 
-gitea.overrideAttrs (old: rec {
+let
+  tags = lib.optional pamSupport "pam"
+    ++ lib.optional sqliteSupport "sqlite sqlite_unlock_notify";
+  tagsString = lib.concatStringsSep " " tags;
+in
+
+buildGoPackage rec {
   pname = "forgejo";
-  version = "1.18.0-rc1-1";
+  version = "1.18.0-1";
 
   src = fetchurl {
     name = "${pname}-src-${version}.tar.gz";
     # see https://codeberg.org/forgejo/forgejo/releases
-    url = "https://codeberg.org/attachments/976c426a-3e04-49ff-9762-47fab50624a3";
-    hash = "sha256-kreBMHlMVB1UeG67zMbszGrgjaROateCRswH7GrKnEw=";
+    url = "https://codeberg.org/attachments/c829784c-3b85-4996-9dc6-09e12e40a93a";
+    hash = "sha256-42bR18T5ATVyhPejeHpOTkeP6VwYu5HFfei8oMjRJy8=";
```

```suggestion
    url = "https://codeberg.org/attachments/5d59ec04-9f29-4b32-a1ef-bec5c3132e26";
    hash = "sha256-RLShwdx8geyFr1Jk5qDVbsEt2hCjdrwX0lNHea7P+pk=";
```
for [v1.18.2-0](https://codeberg.org/forgejo/forgejo/releases/tag/v1.18.2-0)